runtime/dotnet/azurefunctions/Startup.cs

```diff
@@ -211,5 +208,10 @@ private string GetRootDialog(string folderPath)
 
             throw new Exception($"Can't locate root dialog in {dir.FullName}");
         }
+
+        private bool ConfigSectionValid(string val)
```

Comment:
This method doesn't access member state, so it can be made static

```suggestion
        private static bool ConfigSectionValid(string val)
```